packages/epics/src/people/hooks/use-uploadthing-file-uploader.ts

```
@@ -0,0 +1,58 @@
+      Authorization: `Bearer ${jwt}`,
+      'Content-Type': 'application/json',
+    },
+  });
+
+  const [isUploading, setIsUploading] = useState(false);
+  const [uploadedFile, setUploadedFile] = useState<string | null>(null);
+
+  const handleUploadComplete = (res: { url: string }[]) => {
+    console.log('Files: ', res);
+
+    if (res && res[0]?.url) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)